src/extensions/extension-loader/extension-loader.ts

```diff
@@ -339,6 +348,8 @@ export class ExtensionLoader {
     }, {
       fireImmediately: true,
     });
+
+    return Promise.all(bundledLoaded);
```

Comment:
`autoInitExtensions()` method shouldn't return only specific promises with bundled extensions, it's more generic by its name? Maybe we can return all loaded extensions and then filter them some other place?

Reply:
good point. fixed.